lib/Controller/MessagesController.php

```diff
@@ -219,6 +241,18 @@ public function getBody(int $id): JSONResponse {
 		$json['databaseId'] = $message->getId();
 		$json['isSenderTrusted'] = $this->isSenderTrusted($message);
 
+		$sMime = ['isSigned' => false];
+		try {
+			$parsedMessage = Horde_Mime_Part::parseMessage($rawMessage, ['no_body' => true]);
+			if ($parsedMessage->getType() === 'multipart/signed') {
+				$sMime['isSigned'] = true;
+				$sMime['signatureIsValid'] = $this->sMimeService->verifyMessage($rawMessage);
+			}
+		} catch (Horde_Mime_Exception $e) {
+			$this->logger->warning('Failed to parse MIME message', ['error' => $e]);
```

Comment:
```suggestion
			$this->logger->warning('Failed to parse MIME message: ' . $e->getMessage(), ['error' => $e]);
```

Reveals the inner exception message when someone reports the bug